### ISSUE TITLE
Improve notification display timing

### ISSUE THREAD

**Issue by claell** [JabRef/jabref#8753]
**Is your suggestion for improvement related to a problem? Please describe.**
When I perform actions in JabRef, sometimes a notification is displayed at the bottom, like:
![grafik](https://user-images.githubusercontent.com/6171135/166505899-425c681c-d93b-4d6f-8c29-f5e8312bc899.png)

Unfortunately, when notifications pile up, it can take a long time for the queue to finally empty (especially when using undo/redo operations intensively). This is because each notification seems to take a specific amount of time and only after that, the next notification in queue is displayed.

A probable side effect of this problem happens when saving a file: First a notification is displayed, that the file is saved. Afterwards, a notification is displayed, that the file has been saved successfully. Since the notification duration is set to about 5 seconds, this is what the user thinks (or I thought at first when using JabRef) it takes to successfully save the file (while in reality, this operation takes less than a second most times)

**Describe the solution you'd like**
Have a better notification system. Immediately displaying a new notification when it is queued. Either by hiding the previous one or by stacking them up on top of each other (or by another solution).

**Additional context**

**Comment by ThiloteE** [comment]
Hey claell, would you mind checking out the newest development version?  #8733 was just merged, which should alleviate a few of the issues! :)

**Comment by claell** [comment]
Ah, thanks for the heads-up. I'll check it out. Looks, like this issue is a duplicate of #7340.

**Comment by Siedlerchr** [comment]
Yeah ;) Was fortunately already reviewed by @calixtus earlier and then myself. So I will close the issue. If you encounter any more issues with it, feel free to reopen it

**Comment by claell** [comment]
So, if somebody of the team can close this as duplicate, I'll open a new issue about the overlap of the Windows task bar (if this is considered to be an issue).

**Comment by claell** [comment]
Sorry, comment overlap. So do we need another issue about the task bar overlapping notifications on Windows?

**Comment by claell** [comment]
@Siedlerchr I will create a new issue now for the overlapping notifications on Windows: #8769
